README.md

```diff
@@ -1,31 +1,34 @@
 # tt-xla
-tt-xla leverages pjrt to provide a hardware and framework independent interface for compilers and runtimes and simplifies the integration of hardware with frameworks. Please see the [blog](https://opensource.googleblog.com/2023/05/pjrt-simplifying-ml-hardware-and-framework-integration.html) for more information about PJRT project. This project is a fork of [iree-pjrt](https://github.com/stellaraccident/iree-pjrt)
+tt-xla leverages [PJRT](https://github.com/openxla/xla/tree/main/xla/pjrt/c#pjrt---uniform-device-api) interface to simplify the integration of Tenstorrent hardware and [compiler](https://github.com/tenstorrent/tt-mlir) with machine learning frameworks. Please see [this](https://opensource.googleblog.com/2023/05/pjrt-simplifying-ml-hardware-and-framework-integration.html) blog post for more information about PJRT project. This project is a fork of [iree-pjrt](https://github.com/stellaraccident/iree-pjrt).
```

Comment:
Perhaps this is a little clearer:
`tt-xla leverages PJRT to integrate JAX (and in the future other frameworks), tt-mlir and Tenstorrent hardware`?
But  the original version is fine too.

Reply:
Makes sense, thanks, edited